docs/changelog/128858.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 128858
+summary: Fix unsupported privileges error message during role and API key crea…
+area: Authorization
+type: bug
```

Comment:
I think this should be an enhancement since it's not a bug, it's just a suboptimal error message.